### PR TITLE
Minor readability fixes and Policy Definitions note.

### PR DESCRIPTION
**Pull request by ChrisKibble**
Made some minor readability changes to the document and also added a note about how to use the local policy store if your organization does not use a central store.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ This article is for IT admins who manage the new OneDrive sync client in a Windo
   
 3. Copy the .adml and .admx files.
 
-4. Paste the .admx file in your domain's Central Store, \\*domain*\sysvol\domain\Policies\PolicyDefinitions (where *domain* is your domain name, such as corp.contoso.com), and the .adml in the appropriate language subfolder (such as en-us). If the PolicyDefinitions folder does not exist, see [How to create and manage the Central Store for Group Policy Administrative Templates in Windows](https://support.microsoft.com/en-us/help/3087759).
+4. Paste the .admx file in your domain's Central Store, \\*domain*\sysvol\domain\Policies\PolicyDefinitions (where *domain* is your domain name, such as corp.contoso.com), and the .adml in the appropriate language subfolder (such as en-us). If the PolicyDefinitions folder does not exist, see [How to create and manage the Central Store for Group Policy Administrative Templates in Windows](https://support.microsoft.com/en-us/help/3087759) or use your local policy store under %windir%\policydefinitions.
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md]:
```suggestion
4. Paste the .admx file in your domain's Central Store, \\*domain*\sysvol\domain\Policies\PolicyDefinitions (where *domain* is your domain name, such as corp.contoso.com), and the .adml in the appropriate language subfolder (such as en-us). If the PolicyDefinitions folder does not exist, see [How to create and manage the Central Store for Group Policy Administrative Templates in Windows](https://support.microsoft.com/help/3087759) or use your local policy store under %windir%\policydefinitions.
```

File: OneDrive/use-group-policy.md [hunk 2]
```diff
@@ -493,7 +493,7 @@ To find the library ID, sign in as a global or SharePoint admin in Office 365, b
 ![The Getting ready to sync dialog box](media/copy-library-id.png)
 
 > [!NOTE]
-> The special characters in this copied string are in UniCode which needs to be converted to ASCII find and replace the following:
+> The special characters in this copied string are in Unicode which needs to be converted to ASCII find and replace the following:
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md]:
```suggestion
> The special characters in this copied string are in Unicode which needs to be converted to ASCII; find and replace the following:
```

File: OneDrive/use-group-policy.md [hunk 3]
```diff
@@ -388,7 +388,7 @@ Enabling this policy sets the following registry key value to 1.
   
 [HKLM\SOFTWARE\Policies\Microsoft\OneDrive]"FilesOnDemandEnabled"="dword:00000001"
  
-Meet Windows and OneDrive sync client requirements and still can't see Files On-Demand option available at "Settings"? Make sure service "Windows Cloud Files Filter Driver" start type is set to 2 (AUTO_START). Enabling this feature sets the following registry key value to 2.
+Meet Windows and OneDrive sync client requirements and still can't see Files On-Demand option available at "Settings"? Make sure the service "Windows Cloud Files Filter Driver" start type is set to 2 (AUTO_START). Enabling this feature sets the following registry key value to 2.
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md]:
```suggestion
Are you meeting Windows and OneDrive sync client requirements and still can't see the Files On-Demand option available at "Settings"? Make sure the service "Windows Cloud Files Filter Driver" start type is set to 2 (AUTO_START). Enabling this feature sets the following registry key value to 2.
```

**Pull request merged**